lighthouse-core/audits/dobetterweb/inspector-issues.js

```diff
@@ -29,10 +29,32 @@ const UIStrings = {
   /* eslint-enable max-len */
   /** Table column header for the types of problems observed in a website, like security or network errors. */
   columnIssueType: 'Issue type',
-  /** The type of an Issue in Chrome DevTools when a resource is blocked due to the website's cross-origin policy. */
+  /** Issue when a resource is blocked due to the website's cross-origin policy. */
   issueTypeBlockedByResponse: 'Blocked by cross-origin policy',
-  /** The type of an Issue in Chrome DevTools when a site has large ads that use up a lot of the browser's resources. */
+  /** Issue when a site has large ads that use up a lot of the browser's resources. */
   issueTypeHeavyAds: 'Heavy resource usage by ads',
+  /** Issue around "Attribution Reporting API" usage. */
+  issueTypeAttributionReporting: 'Attribution reporting',
+  /** Issue when a client hint is incorrectly used. */
+  issueTypeClientHint: 'Client hint incorrectly used',
+  /** Issue related to CORS. */
+  issueTypeCors: 'CORS Problems',
+  /** Issue when a deprecated feature is used. */
+  issueTypeDeprecated: 'Deprecated feature used',
+  /** Issue when federated authentication request fails. */
+  issueTypeFederatedAuthRequest: 'Failed federated authentication request',
+  /** The type of an issue in Chrome DevTools when some generic problem happens. */
+  issueTypeGeneric: 'Generic issue on the front end',
+  /** Issue when the text doesn't have enough contrast. */
+  issueTypeLowTextContrast: 'Low text contrast',
+  /** Issue related to the reduction of information in user-agent strings. */
+  issueTypeNavigatorUserAgent: 'Using information not present in reduced user-agent strings',
+  /** Issue related to the document rendering in quirks mode. */
```

Comment:
let's suggest to the translator that terms like "legacy" are also a valid descriptor. Or maybe we should backtick quirks...

Reply:
(unresolved): do you perhaps have changes you haven't pushed to the branch?